src/graphs.py

```diff
@@ -100,14 +100,15 @@ def add_item(self, item, reset_clipboard=True):
     self.main_window.no_data_label_box.set_visible(False)
     self.item_boxes[key] = ItemBox(self, item)
     self.main_window.list_box.append(self.item_boxes[key])
+    self.item_menu[key] = self.main_window.list_box.get_last_child()
```

Comment:
Maybe a way to not have this stored two times is to store just the ListBoxRow gotten from get_last_child(). If we need the actual ItemBox we could just query get_child()

Reply:
In the latest commit, I removed item_boxes dictionary, and store it directly in the list_box of the main window. As the dictionary itself is kinda redundant, and the ItemBox itself can be retrieved from item_menu using the same key.

So far it seems to work in my initial tests without new issues.

Reply:
Just FYI, this did introduce some bugs and I reverted this removal of item_boxes for now with a seperate commit.